config/initializers/column_definition.rb

```diff
@@ -0,0 +1,5 @@
+class ActiveRecord::ConnectionAdapters::ColumnDefinition
```

Comment:
Could we leave a comment here for future folks to explain why we're providing this customization? Perhaps at some point it could be removed, so we want some kind of trace back. Thanks!

Reply:
So far made comments for this one, next two will be done today. Feel free to commit any changes to it.

Reply:
Let's just also be sure it's clear if we need to remove any of this code in future versions of Rails, Ruby... thanks!

Reply:
@jywarren for this one I would consider just updating your sql. The error happens when using MySQL5.7 and MapKnitter still uses 5.6. The only reason I  discovered this error is bc for macOS installation (either with homebrew or MySQL community server) the default version is 8 and I had not downgraded it yet. So this file will have to be here always in case you ever update the MySQL version

It could be worth it to just fix. If I remember correctly it was the annotations table and there were only 3 errors thrown. And the errors were caused by using DEFAULT NULL which throws error starting from MySQL5.7

I left the file here anyway because likely other people might use a later version by accident and would get the error

Reply:
OK great, so we might leave a comment by the mysql version as configured in MapKnitter, so that if we upgrade it (and seems likely that we will soon) we can know.

OR, we could just change the default to something other than NULL... 

OK, so let's: 

1. change DEFAULT NULL
2. leave comment here with a copy of the error that it solves (and a link back to this PR)
3. look at upgrading mysql version

Once the comment is added (2) would this be ready to merge? Thanks Sasha, great work!

Reply:
Actually just added:

```
# This can also be removed if DEFAULT NULL is removed from all columns;
# Read more at https://github.com/publiclab/mapknitter/pull/323
```

I think we're good although we can open an issue for "changing `DEFAULT NULL` in a new migration" and also for upgrading mysql!

Reply:
So it's not in the migration its in the actual SQL code. Sure, once we merge this, I can open a separate issue about updating DEFAULT NULL and upgrading mysql. If we're going to upgrade we should maybe try to do it before more people start contributing because having multiple versions of MySQL on your computer can create issues and then they are difficult to get rid of. At least that was the experience for me. 

Just need to add the comments for the other 2 and then were good to go!

Reply:
Agreed re upgrading soon!